app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingPrivacyNoticeViewHolder.kt

```diff
@@ -13,7 +13,10 @@ import org.mozilla.fenix.settings.SupportUtils
 class OnboardingPrivacyNoticeViewHolder(view: View) : RecyclerView.ViewHolder(view) {
 
     init {
-        view.header_text.setOnboardingIcon(R.drawable.ic_onboarding_privacy_notice)
+        view.header_text.setOnboardingIcon(
+            R.drawable.ic_onboarding_privacy_notice,
+            R.color.onboarding_card_icon
```

Comment:
I think `OnboardingWhatsNewViewHolder` should also use this `onboarding_card_icon` color. Could you change `setOnboardingIcon` to just always use `onboarding_card_icon` instead of having a second parameter?

Reply:
:heavy_check_mark: done